templates/islandora-solr-metadata-display.tpl.php

```diff
@@ -30,6 +30,14 @@
           </dd>
           <?php $row_field++; ?>
         <?php endforeach; ?>
+        <?php if (!empty($object_views_count_view)): ?>
+          <dt>
+            Object views
```

Comment:
Nah: https://en.wikipedia.org/wiki/Languages_of_Australia: "Australia has no official language, but is largely monolingual with English being the de facto national language."